tests/db_setup.py

```diff
@@ -1,16 +1,26 @@
 import json
 from os.path import dirname, join
 
+from sqlalchemy import orm
+
 from snowexsql.db import get_db, initialize
+from snowexsql.tables import (Campaign, DOI, Instrument, LayerData,
+                              MeasurementType, Observer, Site)
+from snowexsql.tables.site import SiteObservers
+
+# DB Configuration and Session
+CREDENTIAL_FILE = join(dirname(__file__), 'credentials.json')
+DB_INFO = json.load(open(CREDENTIAL_FILE))
+SESSION = orm.scoped_session(orm.sessionmaker())
```

Comment:
Very important to use a scoped_session
https://docs.sqlalchemy.org/en/20/orm/contextual.html#contextual-thread-local-sessions